hathor/cli/run_node_args.py

```diff
@@ -0,0 +1,64 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import List, NamedTuple, Optional
+
+
+class RunNodeArgs(NamedTuple):
```

Comment:
Could you add a description here pointing to `RunNode.create_parser`, where the arguments are defined. So it's easier to see where to add them?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/636/commits/1b1978be1ca0983ea1284e6298c1ea288f9964f9